Updated Grids and List topics (#533)
IgniteUI/indigo-design-docfx: en/components/list-skeleton.md

## List Item Primary Action

> In Sketch, there are 4 interchangeable Skeleton List Item Primary Actions like the standard [List Item](list-item.md) that are listed below available as `Symbol Overrides`. Thanks to Smart Layout in Sketch, they can be adjusted further by setting unnecessary elements to ~No Symbol.

Reviewer: ```suggestion
In Sketch, there are 4 interchangeable Skeleton List Item Primary Actions representing the ones from the standard [List Item](list-item.md) that are listed below and available as `Symbol Overrides`. Thanks to Smart Layout in Sketch, they can be adjusted further by setting unnecessary elements to ~No Symbol.
```

## List Item Secondary Action

> In Sketch, The Secondary Action of the Skeleton List Item covers all scenarios possible with its counterpart from the standard [List Item](list-item.md) available as `Symbol Overrides`. Thanks to Smart Layout in Sketch, it can be adjusted further by setting unnecessary elements to ~No Symbol.

Reviewer: ```suggestion
In Sketch, The Secondary Action of the Skeleton List Item covers all scenarios possible with the variety of options in the standard [List Item](list-item.md). Thanks to Smart Layout in Sketch, it can be adjusted further by setting unnecessary elements to ~No Symbol.
```

|                  |                                                                                                    |                                                                                                                                       |
| ---------------- | -------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
| Text and Icon            | <img class="responsive-img" src="../images/list_skeleton_item_secondary3.png" srcset="../images/list_skeleton_item_secondary3@2x.png 2x" /> |                                                                                                                                       |

## Thumbnail in Adobe XD

> The Thumbnail component in the left Skeleton List Item area in Adobe XD provides 3 interchangeable options like the standard [List Item](list-item.md) via the `Component States` paradigm to let you easily switch them. This makes it possible to create a matching design for each of the Primary Actions in Sketch.

Reviewer: ```suggestion
The Thumbnail component in the left Skeleton List Item area in Adobe XD provides 3 interchangeable options representing the ones from the standard [List Item](list-item.md) via the `Component States` paradigm to let you easily switch between them.
```

> 

Reviewer: add image

## Actions in Adobe XD

> The Actions component that constitutes the right Skeleton List Item area in Adobe XD covers all scenarios possible with its counterpart from the standard [List Item](list-item.md). This makes it possible to create a matching design for each of the Secondary Actions in Sketch.

Reviewer: ```suggestion
The Actions component that constitutes the right Skeleton List Item area in Adobe XD covers all scenarios possible with its counterpart from the standard [List Item](list-item.md).
```

> 

Reviewer: add image
